src/main/resources/application.yml

```diff
@@ -27,6 +27,7 @@ management:
     web:
       exposure:
         include: info
+imageDirectory: kimonoimages
```

Comment:
Maybe it would be useful to give the file size limits. Like:

spring.servlet.multipart.max-file-size=5MB
spring.servlet.multipart.max-request-size=10MB